Dockerfile

```diff
@@ -3,7 +3,7 @@ ADD . /build
 WORKDIR /build
 RUN make build
 
-FROM gcr.io/distroless/base:debug AS trento-web
+FROM gcr.io/distroless/base:debug AS telemetry-collector
```

Comment:
Should be call it just `telemetry`? This is how the CI creates the container at least

Reply:
changed